src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -24,13 +24,18 @@ internal class RequestTracingConstants
         public const string HostTypeKey = "Host";
         public const string FilterTypeKey = "Filter";
         public const string EnvironmentKey = "Env";
+        public const string FeatureManagementVersionKey = "FeatureManagementVersion";
```

Comment:
This goes on the wire. Can we have something shorter, for example, FMver and FMANver?